modules/hal_nordic/nrf-regtool/nrf-regtoolConfigVersion.cmake

```diff
@@ -4,6 +4,8 @@
 find_program(NRF_REGTOOL nrf-regtool)
 
 if(NRF_REGTOOL)
+  # Use in-tree devicetree Python package
+  set(ENV{PYTHONPATH} ${ZEPHYR_BASE}/scripts/dts/python-devicetree/src)
```

Comment:
To avoid the small risk with polluting the CMake environment, I'd suggest confining this to the relevant command:
https://github.com/zephyrproject-rtos/zephyr/blob/9207def7dac27fb0017b71ad48f7e6c2c3bd29f4/modules/hal_nordic/nrf-regtool/nrf-regtoolConfig.cmake#L14-L15

by inserting:
```cmake
execute_process(
  COMMAND
  ${CMAKE_COMMAND} -E env PYTHONPATH=${ZEPHYR_BASE}/scripts/dts/python-devicetree/src
  ${NRF_REGTOOL} ...
```

Reply:
thx, adjusted

Reply:
Is it necessary for `nrf-regtool --version` too?

Reply:
no, you're right